### ISSUE TITLE
Geolocation for miner and sharder.

### ISSUE THREAD

**Issue by Sriep** [0chain/0chain#793]
Geolocation fields from https://github.com/0chain/0chain/issues/815 added to Event database. Make sure geolocation fields are emitted when miner and sharder geolocation settings change.

New event database endpoint to return the longitude values for miners.
```sql 
select longitude, latitude from Miner
````

New event database endpoint to return the longitude values for sharders.
```sql 
select longitude, latitude from Sharder
````

**Comment by Sriep** [comment]
Reference https://github.com/0chain/0chain/issues/687 split 0chain.yaml
Miner and sharder tables https://github.com/0chain/0chain/issues/790

**Comment by ssardana96** [comment]
this will be done with ticket #790

**Comment by ma2b0043** [comment]
@Sriep @ssardana08 when can we expect to close this issue?

**Comment by iamrz1** [comment]
Closed by [ #1124](https://github.com/0chain/0chain/pull/1124)
